tests/Testsuite.ml

```diff
@@ -17,5 +17,28 @@
 *)
 
 open Scilla_test.Util
+open Core
 
-let () = run_tests [ TestCodegenExpr.All.tests; TestCodegenContr.All.tests ]
+let llvm_version () =
+  let open Build_info.V1 in
```

Comment:
I think this is the simplest way to get the version of the external library in the current switch. It uses `Build_info` module dynamically generated by dune during the build process.

Two other possible solutions:
* Use `opam-client` and `opam-core` instead. Actually, this is the same idea, but the implementation will be more verbose.
* Write a small OCaml library that gets the LLVM version from system headers using C FFI. I think, this could be better in performance, but it will complicate build process (the user must additionally use `opam pin <library repo>`).

The correct way to get rid of this hack is to suggest a patch to LLVM that adds version number to OCaml bindings. The version defined in Golang bindings [exactly by this way](https://github.com/llvm/llvm-project/blob/d4abdd2e3dda8f94219879501d22361e0549592b/llvm/bindings/go/llvm/version.go#L20).
But we'll need to wait for the LLVM 16 release (~2 yrs).